SingInTests/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SingInTests
+{
+    internal class SignInTest
+    {
+        private CreateUserRequest request = new CreateUserRequest
+        {
+            FirstName = "Test",
+            LastName = "Test",
+            Birthday = DateTime.Today.AddYears(-20),
+            Email = "testUser@mail.ru",
+            Password = "password"
+        };
+        private static void Main(string[] args)
+        {
+            var test = new SignInTest();
+            try
+            {
+                test.TryToSingUp();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("User was created");
+            }
+
+            if (test.TryToSingIn())
+            {
+                Console.WriteLine("Successful");
+            }
+            else
+            {
+                Console.WriteLine("Something was wrong");
+            }
+        }
+
+        private async Task TryToSingUp()
```

Comment:
Pull request named sign in tests. But you write logic for sign up. Something went wrong :)

Reply:
How can I log in if the user does not exist in the database?

Reply:
It is logical :)